source/resources/links/feed.xml.builder

```diff
@@ -0,0 +1,30 @@
+xml.instruct!
+xml.feed "xmlns" => "http://www.w3.org/2005/Atom" do
+  site_url = config[:tech_docs][:host]
+  xml.title "UK Government API and Data Exchange Community Slack Community Links"
+  xml.subtitle "Links of interest, being discussed by the UK Government API Community on https://ukgovtapidx.slack.com/"
+  xml.link "href" => URI.join(site_url, current_page.path), "rel" => "self"
+  xml.updated(links.links.first.date_added.to_time.iso8601) unless links.links
+  xml.author do
+    xml.name "UK Government API Community"
+    xml.uri config[:tech_docs][:host]
+    xml.email "api-programme@digital.cabinet-office.gov.uk"
+  end
+
+  links.links.each do |link|
+    tracking_url = append_querystring(link.url, {
+      utm_source: "api.gov.uk",
+      utm_medium: "rss",
+    })
+    content = <<-DOC
+    <a href="#{tracking_url}">#{link.url}</a> was shared in the <a href="https://ukgovtapidx.slack.com/">UK Government API and Data Exchange Community Slack</a>
+    DOC
+    xml.entry do
+      xml.title link.title
+      xml.link "rel" => "alternate", "href" => link.url
+      xml.id link.url
```

Comment:
Commit should try and document why this is the ID

Reply:
This may want to be:
```suggestion
      xml.id "#{link.url}-#{link.date_added.to_s}"
```

So there's a unique ID in case we add the same link multiple times

Reply:
Do we want to be able to send the same link twice?

Reply:
There's things like the API Management / GraphQL guidance that's appeared a couple of times, which I wouldn't say are necessarily the same as other links, but there could be times in the future we re-share something as it's evergreen?